doc/build/core/engines.rst

```diff
@@ -200,13 +200,23 @@ More notes on connecting to MySQL at :ref:`mysql_toplevel`.
 Oracle
 ^^^^^^^^^^
 
-The Oracle dialect uses cx_oracle as the default DBAPI::
+The preferred Oracle Database dialect uses the python-oracledb driver as the
+DBAPI::
 
-    engine = create_engine("oracle://scott:tiger@127.0.0.1:1521/sidname")
+      engine = create_engine(
+          "oracle+oracledb://scott:tiger@127.0.0.1:1521/?service_name=freepdb1"
+      )
 
-    engine = create_engine("oracle+cx_oracle://scott:tiger@tnsname")
+      engine = create_engine("oracle+oracledb://scott:tiger@tnsalias")
 
-More notes on connecting to Oracle at :ref:`oracle_toplevel`.
+For historical reasons, the Oracle dialect uses the obsolete cx_Oracle driver
```

Comment:
**Michael Bayer** (zzzeek) wrote:

OK good, good way to handle this

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/5589